### ISSUE TITLE
sh: pause: command not found

### ISSUE THREAD

**Issue by kozec** [jimmyazrael/NierAutoModSave#3]
At end of script, which appears to work fine, error message is displayed:

```
... snip ...
Finished. Glory to mankind!
Press any key to exit...
sh: pause: command not found
```

It's definitely not fine and at least prefixing it with `if os.name == 'nt':` would be in place :)

**Comment by jimmyazrael** [comment]
Thx! You have a good point. :D

**Comment by jimmyazrael** [comment]
Fixed with commit 5712573de4815ab0aad8a2457b10a64e5a993557

**Comment by kozec** [comment]
Great. By the way, it really worked fine and restored my saves from old Windows installation.

Thank you very much :)
